core/src/main/java/org/elasticsearch/search/aggregations/AggregatorFactory.java

```diff
@@ -53,6 +53,14 @@ public AggregatorFactory(String name, String type) {
         this.type = type;
     }
 
+    public void init(AggregationContext context) {
```

Comment:
Also I don't know how easy it is, but if we pass the aggregation context in the init() method, we might want to remove this parameter from the create() method